package.json

```diff
@@ -73,12 +68,9 @@
     "sanitize-filename": "^1.6.3",
     "send": "0.17.1",
     "sockette": "^2.0.6",
-    "source-map-support": "0.5.19",
```

Comment:
I'm not sure if this one is transitively used by Webpack and might need to be added back as a devDependency